app/controllers/groups/members_controller.rb

```diff
@@ -2,9 +2,8 @@
 
 module Groups
   # Controller actions for Members
-  class MembersController < ApplicationController
+  class MembersController < Groups::ApplicationController
     include MembershipActions
-
     layout 'groups'
```

Comment:
Since you are using `Groups::ApplicationController` this layout line can be removed.

Reply:
Updated in 6dc4a9d5114f8c5f8ef460d3b5108c373769273e